pkg/runtime/pubsub/bulkpub.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
bulkpub.go and bulksub.go contain the default implementation for bulk pub/sub, used when the components do not implement `BulkPublisher` or `BulkSubscriber` interfaces.

Reply:
Not necessarily now, but later we can rename bulkpub /bulksub classes to reflect that they are default implementations. It will help differentiate current classes like bulksub v/s bulk_subscriber.

Reply:
Done - changed to `default_bulksub` and `default_bulkpub`.